src/pages/settings/routes.tsx

```diff
@@ -17,9 +17,14 @@ import * as Settings from './index';
 export const settingsRoutes = (
   <Route path="/settings">
     <Route path="" element={<Settings.Settings />} />
-    <Route path="company_details">
-      <Route path="" element={<Settings.CompanyDetails />} />
+    <Route path="company_details" element={<Settings.CompanyDetails />}>
+      <Route path="" element={<Settings.Details />} />
+      <Route path="details" element={<Settings.Details />} />
+      <Route path="address" element={<Settings.Address />} />
+      <Route path="logo" element={<Settings.Logo />} />
+      <Route path="defaults" element={<Settings.Defaults />} />
       <Route path="documents" element={<Settings.CompanyDocuments />} />
+      <Route path="custom_fields" />
```

Comment:
@turbo124 do we need custom fields tab here? It wasn't there before.

Reply:
> @turbo124 do we need custom fields tab here? It wasn't there before.

David mentioned me on the Jira that he wants to add this tab same as on the Flutter app. So, I just prepared everything for doing this piece of task. I will ping him to give me more instructions for this tab.

Reply:
@beganovich I can't see where in the app we show company custom fields currently, so i advised @Civolilah to put them here